homeworks/serhii.ripchanskyi_Ripchanskiy/11-dependency-injection/vechicle/src/app/components/vehicle/vehicle/vehicle.component.html

```diff
@@ -0,0 +1,20 @@
+<li class="vehicle-list__vehicle-item">
+  <a class="vehicle-list__vehicle-item-link"
+     target="_blank"
+     #el href="{{ vehicle | transformWebsite }}"
```

Comment:
All your properties are moved on the new line that correct! You're right! But on this specific line you have two properties on the same line.

**Suggestion:** split it to two lines, please